database/README.md

```diff
@@ -8,6 +8,20 @@ Some tables only store data emitted via smart contract events. Because we only h
 [CoWSwapEthFlow](https://github.com/cowprotocol/ethflowcontract/blob/main/src/CoWSwapEthFlow.sol) we actually deployed twice so events related to the staging environment should only show up in the staging DB and likewise for production.
 It's also important to note that we only index events from blocks that we are certain will not get reorged. That means specifically that events will be indexed with a block delay of at least 64.
 
+### app\_data
+
+Associates the 32 bytes contract app data with the corresponding full app data.
+
+See [here](https://github.com/cowprotocol/services/issues/1465) for more details. In this table the contract app data is either the old unixfs based scheme, or the new keccak scheme. The new scheme can be validated by keccak-256 hashing the full app data, which should produce the contract app data. The old scheme cannot be validated.
+
+Column             | Type  | Nullable | Details
+-------------------|-------|----------|-------
+ contract_app_data | bytea | not null | 32 bytes. Referenced by `orders.app_data`.
+ full_app_data     | bytea | not null | Is utf-8 but not stored as string because the raw bytes are important for hashing.
+
+Indexes:
+- "app_data_pkey" PRIMARY KEY, btree (contract_app_data)
```

Comment:
nit: Unescaped `_` can result in _italics_ in some markdown renderers.
<img width="778" alt="Screenshot 2023-06-08 at 21 50 34" src="https://github.com/cowprotocol/services/assets/19190235/cd79f05a-b199-4963-816b-718b71f68cd4">